packages/bruno-electron/src/app/watcher.js

```diff
@@ -101,7 +101,7 @@ const addEnvironmentFile = async (win, pathname, collectionUid, collectionPath)
 
     file.data = bruToEnvJson(bruContent);
     file.data.name = basename.substring(0, basename.length - 4);
-    file.data.uid = getRequestUid(pathname);
+    file.data.uid = generateUidBasedOnHash(pathname);
```

Comment:
this is needed! otherwhise the env wil have a diferent uid every startup.
The env files dont are moved without move the collection so the problem solved with requestUid dont apply for envs